### ISSUE TITLE
git pull is for current branch

### ISSUE THREAD

**Issue by YueLinHo** [progit/progit2#531]
book\02-git-basics\sections\remotes.asc line 117

```
If you have a branch set up to track a remote branch 
(see the next section and <<_git_branching>> for more information), 
you can use the `git pull` command to automatically fetch 
and then merge a remote branch into your current branch.(((git commands, pull)))
```

Should be?

```
If your current branch is set up to track a remote branch 
(see the next section and <<_git_branching>> for more information), 
you can use the `git pull` command to automatically fetch 
and then merge that remote branch into your current branch.(((git commands, pull)))
```

----

`If you have a branch set up`

changes to

`If your current branch is set up`

----

`and then merge a remote branch`

to

`and then merge that remote branch`

**Comment by ben** [comment]
I like it. Want to submit a PR?

**Comment by YueLinHo** [comment]
Fixed in PR #532.
